nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -113,6 +113,8 @@ Available as [services.patroni](options.html#opt-services.patroni.enable).
 
 - [Listmonk](https://listmonk.app), a self-hosted newsletter manager. Enable using [services.listmonk](options.html#opt-services.listmonk.enable).
 
+- [Uptime Kuma](https://uptime.kuma.pet/), a fancy self-hosted monitoring tool. Available as [services.uptime-kuma](#opt-services.uptime-kuma.enable).
```

Comment:
This should have probably been put at some point random within this list so there aren't conflicts with other PRs etc, see the comment within the file below this line

Reply:
Yup thanks for the advice, I found out my mistake later on